src/components/standup/index.tsx

```diff
@@ -0,0 +1,151 @@
+import { FC, useState, useEffect } from 'react';
+import moment from 'moment';
+import styles from '@/components/standup/standupContainer.module.scss';
+
+import {
+    useAddStandupMutation,
+    useUserStandupDetailsQuery,
+} from '@/app/services/standup';
+import { useGetUserQuery } from '@/app/services/userApi';
+
+import FormInputComponent from './FormInputComponent';
+import { standupUpdateType } from '@/types/standup.type';
+import { getTotalMissedUpdate } from '@/utils/getTotalMissedUpdate';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+
+const StandUpContainer: FC = () => {
+    const [standupUpdate, setStandupUpdate] = useState<standupUpdateType>({
+        type: 'user',
+        completed: '',
+        planned: '',
+        blockers: '',
+    });
+
+    const [buttonDisable, setButtonDisable] = useState<boolean>(true);
+    const [addStandup] = useAddStandupMutation();
+    const { data: user } = useGetUserQuery();
+    const { data: userStandupdata } = useUserStandupDetailsQuery(user?.id);
+
+    const { SUCCESS, ERROR } = ToastTypes;
+    const standupDates = userStandupdata?.data?.map((element) => element.date);
+    const totalMissedUpdate = getTotalMissedUpdate(standupDates || []);
+    const yesterdayDate = moment().subtract(1, 'days').format('MMMM DD, YYYY');
+
+    const buttonStyleClass = buttonDisable
+        ? `${styles.nonActiveButton}`
+        : `${styles.activeButton}`;
+
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setStandupUpdate((prevStandupUpdate) => ({
+            ...prevStandupUpdate,
+            [event.target.name]: event.target.value,
+        }));
+    };
+    const isValidate = () => {
+        return (
+            standupUpdate.completed !== '' &&
+            standupUpdate.planned !== '' &&
+            standupUpdate.blockers !== ''
+        );
+    };
+    useEffect(() => {
+        const isValid = isValidate();
+        console.log(isValid);
+        setButtonDisable(!isValid);
+    }, [
+        standupUpdate.completed,
```

Comment:
since your `standupUpdate` is an state object will it not update if you just use standupUpdate instead doing
standupUpdate.completed, .planned, .blockers  ?

Reply:
fixed